src/api/ApiSpec.php

```diff
@@ -157,6 +158,18 @@ class ApiSpec {
                 'previousGameId' => 'number',
             ),
         ),
+        'acceptGame' => array(
+            'mandatory' => array(
+                'gameId' => 'number',
+            ),
+            'permitted' => array(),
+        ),
+        'rejectGame' => array(
+            'mandatory' => array(
+                'gameId' => 'number',
+            ),
+            'permitted' => array(),
+        ),
```

Comment:
It doesn't make sense to me that `acceptGame` and `rejectGame` should be two different API functions.  Why is this different from e.g. `reactToAuxiliary`, where something has been offered to you and you have to decide whether or not to do it (using the same API function either way with a different argument)?  I'd call this function something like `reactToOfferedGame` and make it behave like the other react guys.  At the BMInterface level, you're implementing it that way anyway (i.e. the function is `save_join_game_decision()` regardless), so why have two functions at the API level?